src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+  const arr = sourceString.replace(/\s+/g, ' ').split(';');
+
+  for (let i of arr) {
+    i = i.trim();
+
+    if (!i) {
+      arr.slice(i, 1);
```

Comment:
```suggestion
      arr.slice(0, 1);
```
цей блок спрацьовує тільки коли  i = "", тоді arr.slice(i, 1) це arr.slice("", 1), а спрацьовує всерівно як arr.slice(0,1)